README.md

```diff
@@ -0,0 +1,24 @@
+# SDCTA-Data-Visualization-Website
+Install dependencies then run the backend:
+```bash
+$ cd server
+$ npm install
+$ npm start
+```
+
+In a separate terminal install dependencies and run the frontend:
+```bash
+$ cd SDCTA
+$ npm install
+$ npm start
+```
+Note: You will need a .env file that defines DATABASE_URL.
```

Comment:
Maybe make a quick reference to the README in the `server/` folder

Reply:
Not exactly sure what you mean by this, do you mean make a comment in a file in the server folder?

Reply:
There's an existing README in the server/ folder that teaches you how to create a database user and the .env file that you need to run the backend

Reply:
Ah, forgot about that, thank you for letting me know. Updated the README